apparelo/apparelo/doctype/dc/dc.py

```diff
@@ -38,30 +36,28 @@ def on_submit(self):
 		msgprint(_("{0} created").format(comma_and("""<a href="#Form/Purchase Order/{0}">{1}</a>""".format(new_po.name, new_po.name))))
 		msgprint(_("{0} created").format(comma_and("""<a href="#Form/Stock Entry/{0}">{1}</a>""".format(stock_entry.name, stock_entry.name))))
 
-def create_purchase_order(self,abbr):
+def create_purchase_order(self):
```

Comment:
This function is outside the class. Please do not pass and read `self` here. It may confuse.